src/Wallabag/CoreBundle/Command/InstallCommand.php

```diff
@@ -415,14 +421,47 @@ private function isDatabasePresent()
 
     /**
      * Check if the schema is already created.
-     * If we found at least one table, it means the schema exists.
-     *
-     * @return bool
+     * We use the Doctrine Migrations table for the check.
      */
-    private function isSchemaPresent()
+    private function isSchemaPresent(): bool
+    {
+        $schemaManager = $this->entityManager->getConnection()->createSchemaManager();
+
+        return $schemaManager->tablesExist([$this->tableMetadataStorageConfiguration->getTableName()]);
+    }
+
+    private function dropWallabagSchemaOnly(): void
     {
+        $this->runCommand('doctrine:schema:drop', ['--force' => true]);
+
+        $connection = $this->entityManager->getConnection();
+        $databasePlatform = $connection->getDatabasePlatform();
         $schemaManager = $this->entityManager->getConnection()->createSchemaManager();
 
-        return \count($schemaManager->listTableNames()) > 0 ? true : false;
+        try {
+            $schemaManager->dropTable($databasePlatform->quoteIdentifier($this->tableMetadataStorageConfiguration->getTableName()));
+        } catch (TableNotFoundException $exception) {
+        }
```

Comment:
I discovered that I could empty the migration table by running `doctrine:migrations:version --all --delete` so I will change this, so that injecting the Doctrine service is not needed anymore.
also, will rebase on top of #7151 so that it's ready to be merged after :)

Reply:
as I use the Doctrine migration table name to check for the existence of the schema, I need the service anyway, so I'll keep it as is